knowledge-base/common-kb-stackoverflowexception-editing-circular-references.md

```diff
@@ -0,0 +1,63 @@
+---
+title: How to Resolve StackOverflowException When Editing Items with Circular References in ListView for Blazor
+description: This article describes how to avoid a StackOverflowException by modifying object properties to prevent circular references when editing items in the ListView, Grid or TreeList for Blazor.
+type: troubleshooting
+page_title: How to Edit Items with Circular References in ListView for Blazor Without Encountering StackOverflowException
+slug: common-kb-stackoverflowexception-editing-circular-references
+tags: telerik, blazor, listview, grid, treelist, gantt,  stackoverflowexception, circular reference, editing
+res_type: kb
+ticketid: 1660663, 1592634
+---
+
+## Environment
+
+<table>
+    <tbody>
+        <tr>
+            <td>Product</td>
+            <td>ListView for Blazor, <br/> Grid for Blazor, <br /> TreeList for Blazor, <br />  Gantt for Blazor, <br /> Scheduler for Blazor</td>
```

Comment:
The spacing here is weird. There seem to be double spaces and extra spaces between the line breaker and the text.
```suggestion
            <td>ListView for Blazor, <br/>Grid for Blazor, <br />TreeList for Blazor, <br />Gantt for Blazor, <br />Scheduler for Blazor</td>
```